src/ms-review.tex

```diff
@@ -0,0 +1,1786 @@
+        The actual events and instrument response functions can be accessed
+        as attributes on the \code{Observation} object, such as \code{.events}
+        or \code{.aeff} for the effective area information.
+    }
+	\label{fig*:minted:gp_data}
+\end{figure}
+%
+
+\subsection{gammapy.irf}
+\label{ssec:gammapy-irf}
+%
+%\begin{figure}
+%	%	\import{code-examples/generated/}{gp_makers}
+%
+%	\caption{In Fig.~\ref{ig*:minted:irf_examples}, we show some example of Aeff,
+%		PSF, Edisp and BKG read and plotted from a typical \irf.}
+%	\label{ig*:minted:irf_examples}
+%\end{figure}
+%
+The \code{gammapy.irf} sub-package contains all classes and functionality
+to handle IRFs in a variety of formats.
+Usually, \irfs store instrument properties in the form of multi-dimensional
+tables, with quantities expressed in terms of energy (true or reconstructed),
+off-axis angles or cartesian detector coordinates. The main information stored in
+the common \gammaray \irfs are the effective area (Aeff), energy dispersion
+(Edisp), point spread function (PSF) and background rate (BKG). The \code{gammapy.irf}
+sub-package can open and access specific \irf extensions,
+interpolate and evaluate the quantities of interest on both energy and spatial
+axes, convert their format or units in different kinds, plot or write them into
+output files. In the following, we list the main classes of the
+sub-package:
+
+\paragraph{Effective area:}
+\gammapy provides the class \code{EffectiveAreaTable2D} to
+manage Aeff, which is usually defined in terms of true energy and offset angle.
+The class functionalities offer the possibility to read from files or to create
+it from scratch. {\code{EffectiveAreaTable2D} can also convert, interpolate,
+write, and evaluate the effective area for a given energy and offset angles, or
+even plot the multi-dimensional Aeff table.
+
+
+\paragraph{Point spread function:}
+\gammapy allows user to treat different kinds of PSFs,
+in particular, parametric multi-dimensional gaussians (\code{EnergyDependentMultiGaussPSF})
+or King profile functions (\code{PSFKing})
+one. The \code{EnergyDependentMultiGaussPSF} class is able to handle up to three
+gaussians, defined in terms of amplitudes and sigma given for each true energy
+and offset angle bin. Similarly, \code{PSFKing takes} into account the gamma and
+sigma parameters as defined here. The \code{ParametricPSF} allows to create a
+PSF with a representation different from gaussian(s) or King profile(s).
+Finally, the user can take advantage from the \code{PSFMap} class, which creates
+a multi-dimensional map of the PSF in WCS coordinates. At each position, a PSF
+kernel map (\code{PSFKernel}) provides a PSF as a function of the true energy.
+The creation of PSF kernel maps, where the PSF is defined for each
+sky-position, is also given to the user. The latter two can speed up analyses.
+
+\paragraph{Energy dispersion:}
```

Comment:
There is no notion of probability density as mentioned in section 2. To be precised..
And, Edisp should be stored as probability or we renormalise it internally?